chatbot-conversational_AI/bot-observer-tool/app/server.js

```diff
@@ -0,0 +1,159 @@
+const express = require('express');
+const     sql = require('mssql');
+const     got = require('got');
+const     app = express();
+
+
+/********************************************************
+ *  main API endpoint                                   *
+ ********************************************************/
+
+app.get('/', (req, res) => {
+  res.send({
+    message: 'Here you can see which API endpoints are available',
+    endpoints: [
+      {method: 'GET', path: '/bot', description: 'Shows the whole knowledge of the bot. It lists all answers and corresponding questions.'},
+      {method: 'GET', path: '/botcredentials', description: 'Shows the automatically requested credentials for SAP CAI'},
+      {method: 'GET', path: '/stack', description: 'Lists all Questions from Stack that are tagged with [kyma-runtime].'},
+      {method: 'GET', path: '/db', description: 'Shows the linking database that stores the questions from Stack and the related bot knowledge.'}
+    ],
+  });
+});
+
+
+
+/********************************************************
+ *  Bot Data - Endpoints '/bot' and '/botcredentials'   *
+ ********************************************************/
+
+var cai_credentials = null;
+ 
+const cai_credentials_url = 'https://sapcai-community.authentication.eu10.hana.ondemand.com/oauth/token';

```

Comment:
do not hardcode credentials and urls. Use them from a environment variable injected via secret